tests/apm/mod.rs

```diff
@@ -13,7 +13,7 @@ fn timed_query(_client: Client, command_result: &CommandResult) {
     };
 
     // Sanity check
-    assert!(duration >= 1500000000);
+    assert!(duration >= 500000000);
```

Comment:
Actually, the failing assertion might not be just caused by this. But it still doesn't seem like it was necessary to change.